templates/pages/page.html.twig

```diff
@@ -276,7 +276,7 @@
                                                     <p>{{ page.content.contact_form_form_sub_heading }}</p>
                                                 {% endif %}
 
-                                                <form id="js-contact-form" class="govuk-!-width-three-quarters" action="/{{slug}}" method="post">
+                                                <form id="js-contact-form" class="govuk-!-width-three-quarters pardot-submit" action="/{{slug}}" method="post">
```

Comment:
What is this doing here?

Reply:
This is the form used on the different pillar pages. The 'pardot-submit' is used in pardot.js to target the form which sends the data to a Pardot controller. It as per the document for Pardot here https://github.com/Crown-Commercial-Service/ccs-frontend/blob/development/docs/pardot.md (under **submitting user's email address from a form** heading)